src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -60,68 +60,43 @@ def create_launcher(fluent_launch_mode: LaunchMode = None, **kwargs):
     """
     _process_invalid_args(kwargs["dry_run"], fluent_launch_mode, kwargs)
     if fluent_launch_mode == LaunchMode.STANDALONE:
-        return StandaloneLauncher(
-            mode=kwargs["mode"],
-            ui_mode=kwargs["ui_mode"],
-            graphics_driver=kwargs["graphics_driver"],
-            product_version=kwargs["product_version"],
-            version=kwargs["version"],
-            precision=kwargs["precision"],
-            processor_count=kwargs["processor_count"],
-            journal_file_names=kwargs["journal_file_names"],
-            start_timeout=kwargs["start_timeout"],
-            additional_arguments=kwargs["additional_arguments"],
-            env=kwargs["env"],
-            cleanup_on_exit=kwargs["cleanup_on_exit"],
-            start_transcript=kwargs["start_transcript"],
-            case_file_name=kwargs["case_file_name"],
-            case_data_file_name=kwargs["case_data_file_name"],
-            lightweight_mode=kwargs["lightweight_mode"],
-            py=kwargs["py"],
-            gpu=kwargs["gpu"],
-            cwd=kwargs["cwd"],
-            topy=kwargs["topy"],
-            start_watchdog=kwargs["start_watchdog"],
-            file_transfer_service=kwargs["file_transfer_service"],
-        )
+        unsupported_args = ["container_dict", "dry_run", "scheduler_options"]
+        for arg_name in unsupported_args:
+            if arg_name in kwargs:
+                kwargs.pop(arg_name)
+        return StandaloneLauncher(**kwargs)
```

Comment:
We can still pass something like `container=True` which is unsupported and not present in the `unsupported_args` list. So to avoid this we have kept them explicit. i.e without `StandaloneLauncher(**kwargs)`.

Reply:
@raph-luc  We can update it as follows

```
    if fluent_launch_mode == LaunchMode.STANDALONE:
        supported_args = ["mode", "ui_mode", ........]
        for arg_name in kwargs:
            if arg_name not in supported_args :
                kwargs.pop(arg_name)
        return StandaloneLauncher(**kwargs)
```